modules/home/assets/js/components/promotions/sidebar-default.js

```diff
@@ -0,0 +1,41 @@
+import { Box, Paper, Stack, Typography } from '@elementor/ui';
+import Button from '@elementor/ui/Button';
+import List from '@elementor/ui/List';
+import ListItem from '@elementor/ui/ListItem';
+import SideBarCheckIcon from '../../icons/side-bar-check-icon';
+import ListItemText from '@elementor/ui/ListItemText';
+
+const SidebarDefault = ( { header, cta, repeater } ) => {
+	return (
+		<Paper elevation={ 0 } sx={ { p: 3 } }>
+			<Stack gap={ 1.5 } sx={ { alignItems: 'center', textAlign: 'center', pb: 4 } }>
```

Comment:
Can you please try this? We always try to limit the use of the `sx` styling.

```suggestion
			<Stack alignItems="center" textAlign="center" gap={ 1.5 } sx={ { pb: 4 } }>
```

Reply:
I'll check. That was copied as is from original component